lib/logic/session-store.js

```diff
@@ -19,6 +19,24 @@ SessionStore.prototype.getRequestId = function() {
     return this.store.getItem(ID_REQUEST_ID);
 };
 
+SessionStore.prototype.getGithubApiEndpoint = function() {
+    return this.store.getItem("github_api_endpoint") || "https://api.github.com";
+};
+
+SessionStore.prototype.getSupplementaryApiEndpoint = function() {
+    return this.store.getItem("supplementary_api_endpoint") || "https://review.rocks";
+};
+
+SessionStore.prototype.getGithubDelays = function() {
+    // All values are ms
+    var str = this.store.getItem("github_delays");
+    return str ? JSON.parse(str) : {
+        POST_MERGE: 1000,
+        POST_PUSH: 200
```

Comment:
I really really like units to be specified in my constants. Please `POST_MERGE_MS`